tools/packaging/kernel/build-kernel.sh

```diff
@@ -574,6 +584,10 @@ main() {
 				PS4=' Line ${LINENO}: '
 				set -x
 				;;
+			D)
+				dpu_vendor="${OPTARG}"
+				[[ "${dpu_vendor}" == "${GV_NVIDIA}" ]] || die "DPU vendor only support nvidia"
```

Comment:
GV == GPU vendor, let's add a DV for DPU vendor.

Reply:
```
readonly GV_INTEL="intel"
readonly GV_NVIDIA="nvidia"
```

Looks like it's not necessary to add `GV_` prefix to the const variable name, how about replace the prefix to `VENDOR_`:

```
readonly VENDOR_INTEL="intel"
readonly VENDOR_NVIDIA="nvidia"
```
So they can be used for GPU, GPU or other kind of devices.